code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+
+	brow_left
+		name = "Brow Left"
+		icon_state = "brow_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	brow_right
+		name = "Brow Right"
+		icon_state = "brow_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	cheekspot_left
+		name = "Cheek Spot (Left Cheek)"
+		icon_state = "cheekspot_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	cheekspot_right
+		name = "Cheek Spot (Right Cheek)"
+		icon_state = "cheekspot_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	cheshire_left
+		name = "Cheshire (Left Cheek)"
+		icon_state = "cheshire_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	cheshire_right
+		name = "Cheshire (Right Cheek)"
+		icon_state = "cheshire_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	crow_left
+		name = "Crow Mark (Left Eye)"
+		icon_state = "crow_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	crow_right
+		name = "Crow Mark (Right Eye)"
+		icon_state = "crow_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	ear_left
+		name = "Ear Cover (Left)"
+		icon_state = "ear_l"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	ear_right
+		name = "Ear Cover (Right)"
+		icon_state = "ear_r"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	eyestripe
+		name = "Eye Stripe"
+		icon_state = "eyestripe"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyecorner_left
+		name = "Eye Corner Left"
+		icon_state = "eyecorner_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyecorner_right
+		name = "Eye Corner Right"
+		icon_state = "eyecorner_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyelash_left
+		name = "Eyelash Left"
+		icon_state = "eyelash_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyelash_right
+		name = "Eyelash Right"
+		icon_state = "eyelash_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	fullfacepaint
+		name = "Full Face Paint"
+		icon_state = "fullface"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	heterochromia
```

Comment:
I dont know about this one.

Reply:
> Please note that names may not necessarily dictate what they are actually used for in-universe on someone's character.
![image](https://user-images.githubusercontent.com/3950115/65822951-a99d1700-e201-11e9-9aca-8b69515f6bb3.png)

Reply:
It's heterochromia. People will use it to have differently coloured eyes and you can bet your ass it's gonna be an epidemic that nobody wants.

Reply:
It's heterochromia. People will use it to have differently coloured eyes and you can bet your ass it's gonna be an epidemic that nobody wants.

Reply:
This doesn't happen on servers that do have heterochromia.

Reply:
The possibility of everyone suddenly having heterochromia is an understandable concern and I do anticipate everyone suddenly going all out as soon as the ball is dropped and someone tells all their friends.
That's not a guarantee, though, and indeed, it's certainly no epidemic on other servers. Things tend to die down as they're integrated and soon enough it gets to a "reasonable" level, one can argue. Also worth mentioning that having a larger number of entries on the list makes an individual one a little bit more difficult to notice.

And if it's an epidemic, that suggests a lot of people want it. How can no-one then want it?

I've already shown it can be used for other things besides heterochromia iridis/iridum.
In any case, if it's really such a problem, couldn't something be done about it anyway? There's nothing stopping people right now from putting that in their flavour texts and images and indeed people do every now and then.

Would a rename of it be all right?

Reply:
Nah idc just leave it as is, not worth it

Reply:
Nah idc just leave it as is, not worth it

Reply:
I mean...



ok...